From 620cc47b25b5c4dda47f00fddba0cce5b45b9057 Mon Sep 17 00:00:00 2001
From: kitakar5525 <34676735+kitakar5525@users.noreply.github.com>
Date: Thu, 5 Dec 2019 15:03:20 +0900
Subject: [PATCH 2/4] mwifiex: s0ix test: add __maybe_unused to
 mwifiex_pcie_suspend/mwifiex_pcie_resume for 5.4

Add __maybe_unused to old mwifiex_pcie_suspend/mwifiex_pcie_resume
functions to suppress warning on build.

---
 drivers/net/wireless/marvell/mwifiex/pcie.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/wireless/marvell/mwifiex/pcie.c b/drivers/net/wireless/marvell/mwifiex/pcie.c
index 7f4e75fa7..61faf86f4 100644
--- a/drivers/net/wireless/marvell/mwifiex/pcie.c
+++ b/drivers/net/wireless/marvell/mwifiex/pcie.c
@@ -147,7 +147,7 @@ static bool mwifiex_pcie_ok_to_access_hw(struct mwifiex_adapter *adapter)
  * If already not suspended, this function allocates and sends a host
  * sleep activate request to the firmware and turns off the traffic.
  */
-static int mwifiex_pcie_suspend(struct device *dev)
+static int __maybe_unused mwifiex_pcie_suspend(struct device *dev)
 {
 	struct mwifiex_adapter *adapter;
 	struct pcie_service_card *card = dev_get_drvdata(dev);
@@ -190,7 +190,7 @@ static int mwifiex_pcie_suspend(struct device *dev)
  * If already not resumed, this function turns on the traffic and
  * sends a host sleep cancel request to the firmware.
  */
-static int mwifiex_pcie_resume(struct device *dev)
+static int __maybe_unused mwifiex_pcie_resume(struct device *dev)
 {
 	struct mwifiex_adapter *adapter;
 	struct pcie_service_card *card = dev_get_drvdata(dev);
-- 
2.24.0

